elasticsearch/azuredeploy.json

```diff
@@ -37,15 +26,24 @@
         "North Central US",
         "East Asia",
         "Southeast Asia",
-        "West Europe"
+        "West Europe",
+        "North Europe"
       ],
+      "defaultValue": "ResourceGroup",
+      "metadata": {
+        "description": "Location where resources will be provisioned.  A value of 'ResourceGroup' will deploy the resource to the same location of the resource group the resources are provisioned into"
+      }
+    },
+    "storageAccountPrefix": {
+      "type": "string",
+      "defaultValue": "elastic",
       "metadata": {
-        "description": "Location where resources will be provisioned"
+        "description": "Storage account prefix used when creating the storage accounts used for storing virtual machine disks (must be lowercase charactors or numbers and max length of 7)"
```

Comment:
Yeah, I can do that.  Similar requirements will be necessary for any template creating a new storage account (24 character limit), although given the fact that we are appending the unique-string and count it reduces that a bit. A substr() function might be nice to have at some point in the future.